Modules/_hashopenssl.c

```diff
@@ -1107,21 +1107,25 @@ generate_hash_name_list(void)
 /*[clinic input]
 _hashlib.get_fips_mode -> int
 
-Determine the OpenSSL FIPS mode of operation.
+Determine the OpenSSL FIPS mode of operation. For OpenSSL 3.0.0 and newer
+it returns the state of the default provider in the default OSSL context.
+It's not quite the same as FIPS_mode() but good enough for unittests.
```

Comment:
Maybe add:

```suggestion
It's not quite the same as FIPS_mode() but good enough for skipping unittests.
```

Reply:
The function can also be used to test the outcome of a test, e.g. if ``md5()`` fails correctly.

Reply:
Alright, that makes sense.